ncm-metaconfig/src/main/metaconfig/irods/tests/profiles/config.pan

```diff
@@ -0,0 +1,17 @@
+object template config;
+
+include 'metaconfig/irods/config';
+
+bind "/software/components/metaconfig/services/{/etc/nagios/irods/irods_environment.conf}/contents" = irods_environment_client_config;
+
+prefix "/software/components/metaconfig/services/{/etc/nagios/irods/irods_environment.conf}";
+"module" = "jsonpretty";
+
+
+prefix "/software/components/metaconfig/services/{/etc/nagios/irods/irods_environment.conf}/contents";
+
+"irods_host" = "iicat01.ugent.be";
```

Comment:
We should avoid to set such hostname. Something more anonymous would be better.

Reply:
icat01.example.org?